### PR TITLE
Remove qtip2

### PR DESCRIPTION
**Pull request by harshlele**
For #2083 - Commented out all uses of qtip2

below added by bradymiller:
Up For Grabs demo for this PR at:
https://www.open-emr.org/wiki/index.php/Development_Demo#Gamma_-_Up_For_Grabs_Demo

### CODE REVIEW

**Comment by bradymiller** [comment]
Up For Grabs demo for this PR at:
https://www.open-emr.org/wiki/index.php/Development_Demo#Gamma_-_Up_For_Grabs_Demo

**Comment by bradymiller** [comment]
hi @harshlele ,

Another place that uses qtip is here:
https://github.com/openemr/openemr/blob/master/interface/forms/eye_mag/view.php#L183
(if you remove qtip2 from this line then it won't include the qtip2 library there)

Also feel free to just yank the code rather than comment it out. Makes it easier to follow and the goal is to completely remove it anyways :)

thanks, -brady

**Comment by bradymiller** [comment]
hi @ophthal ,
Just wanted to make sure you are ok with this. We are trying to remove qtip2 since it is no longer supported and doesn't work with newer versions of jquery. Native HTML5 tooltips don't look as nice, but seem ok.
thanks,, -brady

**Comment by ophthal** [comment]
Ok what about using bootstrap tooltip?

**Comment by bradymiller** [comment]
hi @ophthal ,
Thanks for the quick reply. Sounds like a good plan to use the bootstrap tooltip.
-brady

File: interface/forms/eye_mag/a_issue.php [hunk 1]
```diff
@@ -1351,24 +1349,18 @@ class="ui-button ui-widget ui-state-default ui-corner-all ui-button-text-only na
...
+} ?>');     
 
         $(document).ready(function() {
-            $('[title]').qtip({
-                position: {
-                    my: 'top Right',  // Position my top left...
-                    at: 'bottom Left', // at the bottom right of...
-                    target: 'mouse' // my target
-                }
-            });
```
Comment (bradymiller) [inline interface/forms/eye_mag/a_issue.php]: Then try following here instead and cross fingers :) :
```
            $('[title]').tooltip();
```

File: interface/forms/eye_mag/js/eye_base.php [hunk 2]
```diff
@@ -2147,32 +2147,23 @@ function showpnotes(docid) {
...
 
-
-                  $('[title]').qtip({
-                                    position: {
-                                    my: 'top Right',  // Position my top left...
-                                    at: 'bottom Left', // at the bottom right of...
-                                    target: 'mouse' // my target
-                                    }
-                                    }
-                                    );
```
Comment (bradymiller) [inline interface/forms/eye_mag/js/eye_base.php]: Also try following here instead and cross fingers :) :
```
            $('[title]').tooltip();
```

File: interface/forms/eye_mag/js/eye_base.php [hunk 3]
```diff
@@ -2147,32 +2147,23 @@ function showpnotes(docid) {
...
                   $('#form_PCP,#form_rDOC').change(function() {
-                                                   update_DOCS();
+                                                   update_DOCS();   
                                                    });
 
                   $('#tooltips_status').html($('#PREFS_TOOLTIPS').val());
                   if ($("#PREFS_TOOLTIPS").val() == "<?php echo xla('Off'); ?>") {
-                  $('[title]').qtip('disable');
```
Comment (bradymiller) [inline interface/forms/eye_mag/js/eye_base.php]:
```
$('[title]').tooltip('disable');
```

File: interface/forms/eye_mag/js/eye_base.php [hunk 4]
```diff
@@ -2147,32 +2147,23 @@ function showpnotes(docid) {
...
+                  
                   }
                   $('#tooltips_toggle,#tooltips_status').click(function() {
                                                                if ($("#PREFS_TOOLTIPS").val() == "<?php echo xla('On'); ?>") {
                                                                $('#PREFS_TOOLTIPS').val('<?php echo xla('Off'); ?>');
                                                                $("#tooltips_status").html('<?php echo xla('are off'); ?>');
-                                                               $('[title]').qtip('disable');
```
Comment (bradymiller) [inline interface/forms/eye_mag/js/eye_base.php]:
```
$('[title]').tooltip('disable');
```

File: interface/forms/eye_mag/js/eye_base.php [hunk 5]
```diff
@@ -2147,32 +2147,23 @@ function showpnotes(docid) {
...
+                                                               
                                                                } else {
                                                                $('#PREFS_TOOLTIPS').val('<?php echo xla('On'); ?>');
                                                                $('#tooltips_status').html('<?php echo xla('are on'); ?>');
-                                                               $('[title]').qtip('enable');
```
Comment (bradymiller) [inline interface/forms/eye_mag/js/eye_base.php]:
```
$('[title]').tooltip('enable');
```

**Comment by bradymiller** [comment]
hi @harshlele ,
Just left some comments that will hopefully get us to using the bootstrap tooltip.
-brady

**Comment by bradymiller** [comment]
hi @harshlele ,
Nice work and thanks for the contribution! Will bring this into codebase now. In order to reproduce the no show/show of tooltips I was able to hack something together (otherwise, if we turn off the tooltips, it was showing them in html5 native form), which will bring in a commit following yours. thanks, -brady

**Comment by bradymiller** [comment]
to follow up:
Here's the commit to make the tooltips turn off and on:
https://github.com/openemr/openemr/commit/f46e7580b7dcba2c6c2f4d18b2f2d0d20ccb23d7
And here's the commit that got rid of all the loose qtip2 pieces (openemr now has zero qtip2 stuff :) ):
https://github.com/openemr/openemr/commit/3097770f6c0eb05993553919eae65d7cf1b43cb0

Thanks for ridding the codebase of qtip2! :)

**Pull request merged**